Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.20-alpine
+
+WORKDIR /app
+
+COPY go.mod ./
+RUN go mod download
+
+COPY ./ ./
+
+WORKDIR /app/cmd/versitygw
+RUN go build -o versitygw
+
+RUN mkdir /tmp/vgw
```

Comment:
Layering here will mean all the stuff done and removed will still be in the final image.